doc/source/provider_video.rst

```diff
@@ -98,7 +98,7 @@ appropriate entry for video options:
     augmentation_config = {"type": "image",
                            "scale": [0.875, 0.875]}
 
-    aeon_config = {"manifest_filename": "train.csv",
+    aeon_config = {"manifest_filename": "train.tsv",
                    "etl": (video_config, label_config),
```

Comment:
why ? tsv we had becouse of typo in benchmark test

Reply:
We have some places using csv and some tsv. CSV suggests commas as separators and is a widely known format, but we use tabs everywhere as separators so tsv made more sense to me.

Reply:
We have some places using csv and some tsv. CSV suggests commas as separators and is a widely known format, but we use tabs everywhere as separators so tsv made more sense to me.

Reply:
ok, you are right